src/hotspot/cpu/riscv/c1_LIRAssembler_riscv.cpp

```diff
@@ -2145,7 +2145,7 @@ void LIR_Assembler::typecheck_lir_store(LIR_OpTypeCheck* op, bool should_profile
 }
 
 void LIR_Assembler::add_debug_info_for_branch(address adr, CodeEmitInfo* info) {
```

Comment:
I see this function is never called: void LIR_Assembler::add_debug_info_for_branch(address adr, CodeEmitInfo* info)
So I would suggest remove this function. Then following two functions introduced by this PR would become unnecessary:
  void relocate(address at, RelocationHolder const& rspec, int format = 0) {
    code_section()->relocate(at, rspec, format);
  }
  void relocate(address at, relocInfo::relocType rtype, int format = 0, jint method_index = 0) {
    code_section()->relocate(at, rtype, format, method_index);
  }

Reply:
Thanks for pointing this out. All done.